core-processor/src/precharge.rs

```diff
@@ -250,10 +250,17 @@ impl ContextCharged<ForOriginalCode> {
     ) -> PrechargeResult<ContextCharged<ForInstrumentedCode>> {
         self.charge_gas(
             PreChargeGasOperation::ModuleInstrumentation,
-            block_config.costs.instrumentation.cost_for_with_bytes(
-                block_config.costs.instrumentation_per_byte,
-                original_code_len_bytes.into(),
-            ),
+            block_config
+                .costs
+                .instrumentation_costs
```

Comment:
etc so usage would look sufficient 
```suggestion
                .instrumentation
```